client/packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx

```diff
@@ -20,7 +20,8 @@ export const getEditableQuantityColumn = <
     const [value, setValue] = useState(quantity);
     const [error, setError] = useState(false);
 
-    const tryUpdateValue = (value: number) => {
+    const tryUpdateValue = (value: number | undefined) => {
+      if (value === undefined) return;
```

Comment:
Don't change the value at all if the user tries to clear it -- seems appropriate for a table input (though this one is not actually used anywhere)